### ISSUE TITLE
[Bug]: Autohotkey: fails to download

### ISSUE THREAD

**Issue by dbobak** [ScoopInstaller/Extras#10270]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

autohotkey

### Expected/Current Behaviour

Installing autohotkey returns 403 Forbidden error. 
But after manually clicking the reported link (`https://www.autohotkey.com/download/2.0/AutoHotkey_2.0.2.zip`), I can download the file.

### Steps to Reproduce

```console
PS C:\Users\deni> scoop install autohotkey
Installing 'autohotkey' (2.0.2) [64bit] from extras bucket
The remote server returned an error: (403) Forbidden.
URL https://www.autohotkey.com/download/2.0/AutoHotkey_2.0.2.zip is not valid
```

### Possible Solution

I'm not sure about the possible solution.

### Scoop and Buckets Version

```console
PS C:\Users\deni> scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'everyx_scoop-bucket' bucket:
643f605 (HEAD -> master, origin/master, origin/HEAD) update README

'extras' bucket:
13978f708 (HEAD -> master, origin/master, origin/HEAD) x64dbg: Update to version 2023-01-18_11-15

'main' bucket:
c39a299f8 (HEAD -> master, origin/master, origin/HEAD) z3: Update to version 4.12.1

'nonportable' bucket:
41e4f2e (HEAD -> master, origin/master, origin/HEAD) spotx-np: Update to version 1.7-20230117-eb5683

'r-bucket' bucket:
2082110b (HEAD -> master, origin/master, origin/HEAD) quarto-prerelease: Update to version 1.3.119

'versions' bucket:
59e742cd3 (HEAD -> master, origin/master, origin/HEAD) everything-alpha: Update to version 1.5.0.1335a
```

### Scoop Config

```console
PS C:\Users\deni> scoop config

last_update         aria2-enabled scoop_branch scoop_repo
-----------         ------------- ------------ ----------
19.01.2023 17:42:50         False master       https://github.com/lukesampson/scoop
```

### PowerShell Version

```console
PS C:\Users\deni> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.3.1
PSEdition                      Core
GitCommitId                    7.3.1
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by HUMORCE** [comment]
Cannot reproduce.

```
❯ scoop download extras/autohotkey
INFO  Downloading 'autohotkey' [64bit] from extras bucket
AutoHotkey_2.0.2.zip (2.9 MB) [===========================================================================================] 100%
Checking hash of AutoHotkey_2.0.2.zip ... ok.
'autohotkey' (2.0.2) was downloaded successfully!

❯ scoop config proxy
'proxy' is not set
```

**Comment by evanratt** [comment]
I am experiencing this issue, as well. Everything else I have tried to install from the main and extras buckets has worked, but autohotkey fails with a 403 error:
```
PS C:\Users\Evan> scoop install autohotkey
Installing 'autohotkey' (2.0.2) [64bit] from extras bucket
The remote server returned an error: (403) Forbidden.
URL https://www.autohotkey.com/download/2.0/AutoHotkey_2.0.2.zip is not valid
``` 

However, if I copy and paste that URL into a browser, AutoHitkey_2.0.2.zip downloads successfully.

### Scoop and Buckets Version
```
PS C:\Users\Evan> scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
cd724b1a2 (HEAD -> master, origin/master, origin/HEAD) openlens: Update to version 6.4.7

'main' bucket:
dfff39ff2 (HEAD -> master, origin/master, origin/HEAD) ttyper: Update to version 1.0.0-pre.2
``` 

### Scoop Config
```
PS C:\Users\Evan> scoop config

last_update          scoop_repo                              scoop_branch
-----------          ----------                              ------------
3/17/2023 5:40:52 PM https://github.com/ScoopInstaller/Scoop master
```

### PowerShell Version
```
PS C:\Users\Evan> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.3.3
PSEdition                      Core
GitCommitId                    7.3.3
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by michaeloyer** [comment]
I just discovered this this morning as well. It looks like this is failing due to protections from Cloudflare on autohotkey.com. At least that's what it looks like if I try to download the file in Powershell: 

```
iwr https://www.autohotkey.com/download/2.0/AutoHotkey_2.0.2.zip
```

I got around this temporarily but updating the manifest file at `~/scoop/buckets/extras/bucket/autohotkey.json` with the version that's over on the [Github releases Page](https://github.com/AutoHotkey/AutoHotkey/releases) like this:

```
{
....
    "url": "https://github.com/AutoHotkey/AutoHotkey/releases/download/v2.0.2/AutoHotkey_2.0.2.zip",
....
}
```

I'm not sure what makes the difference for some over others to get past the Cloudflare protections. Using the github releases page to download the zip from for this to affect fewer people might be something to consider unless there'd be someway to get the site maintainers to turn the down the protections for the downloads page specifically.

Also there may be a better way to do this in scoop without having to touch the manifest file but I didn't see anything in the `scoop update --help` command that looked like I could pass a temporary URI.
